packages/material-ui-system/src/createTheme/createTheme.d.ts

```diff
@@ -0,0 +1,42 @@
+import { Breakpoints, BreakpointsOptions } from './createBreakpoints';
+import { Mixins, MixinsOptions } from './createMixins';
+import { Shadows } from './shadows';
+import { Shape, ShapeOptions } from './shape';
+import { Spacing, SpacingOptions } from './createSpacing';
+import { Transitions, TransitionsOptions } from './createTransitions';
+
+export { Breakpoint, BreakpointOverrides } from './createBreakpoints';
+
+export type Direction = 'ltr' | 'rtl';
+
+export interface ThemeOptions {
+  shape?: ShapeOptions;
+  breakpoints?: BreakpointsOptions;
+  direction?: Direction;
+  mixins?: MixinsOptions;
+  palette?: Record<string, any>;
+  shadows?: Shadows;
+  spacing?: SpacingOptions;
+  transitions?: TransitionsOptions;
+  unstable_strictMode?: boolean;
```

Comment:
This shouldn't be part of a baseline theme. It's specific to the /core package for historical reasons

Reply:
Good catch, I thought I removed it